back/lib/participation_method/proposals.rb

```diff
@@ -6,46 +6,41 @@ def self.method_str
       'proposals'
     end
 
-    def transitive?
-      false
-    end
-
     def assign_defaults_for_phase
       super
+      phase.reacting_dislike_enabled = false
       phase.expire_days_limit ||= 90
       phase.reacting_threshold ||= 300
     end
 
-    # def assign_defaults(_)
-    #   super # TODO: default status and publication status
-    # end
-
     def budget_in_form?(_)
       false
     end
 
-    def proposed_budget_in_form?
-      false
-    end
-
-    # def update_if_published?
-    #   super # TODO: if no reviewing and no reactions + toggle
-    # end
-
     def supports_automated_statuses?
-      true # TODO: This is temorary, until proposal statuses are implemented
+      true
     end
 
-    # def supports_status?
-    #   super # TODO: separate proposal statuses
-    # end
-
     def supports_serializing?(attribute)
       %i[expire_days_limit reacting_threshold].include?(attribute)
     end
 
     def supports_serializing_input?(attribute)
-      %i[expires_at reactions_needed].include?(attribute)
+      %i[expires_at reacting_threshold].include?(attribute)
+    end
+
+    def transitive?
+      false
+    end
+
+    def use_reactions_as_votes?
```

Comment:
Can you give a bit more context on this? What does it mean exactly to use reactions as votes?

Reply:
This means that in proposals, we're reusing the `Reaction` model to implement voting logic (and not baskets, which is what we would normally use to model votes). We chose for this implementation to save time, but in the long run we would prefer not to reuse reactions for this. So in the editing policy for ideas we, don't want to allow editing inputs that have been voted on, and therefore call `use_reactions_as_votes?` to know if we have to look at whether or not reactions are present.